plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -79,7 +84,7 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
     // TODO Come up with a better way than this. This might not be an issue after CON-1069 since we won't be building
     //  the conclave-build image anymore.
     fun buildInDocker(buildInDocker: Property<Boolean>): Boolean {
-        return !OperatingSystem.current().isLinux || buildInDocker.get()
+        return !OperatingSystem.current().isLinux || buildInDocker.get() || runtimeType.get() == GRAMINE
```

Comment:
Building in `conclave-build` should be the default. Only for GraalVM is the `buildInDocker` flag which can turn this off. The reason this flag exists is purely to make sure we haven't accidently prevented existing users have built enclaves in 1.3 from building in 1.4 bc of a Docker issue we didn't encounter. For Gramine this isn't a concern since it's a new feature of 1.4 and thus there are not any existing users to potentially disturb.

Your change here looks correct.

Reply:
Also, for Python the `conclave-build` isn't used, but that's just a prioritisation issue. If we have time before 1.4 to get that working inside `conclave-build` then great, but we don't want to risk 1.4 bc of it.

Reply:
Ok. I made this question because I had previously understood that we should not allow building enclaves (Java and Python) outside the container.
I would keep Python out of the build in the container and revert some changes so that we have conditional code to have `conclave-build` Java enclaves and normal out-of-container build for Python.